src/grpc_argument_validator/validate_args_decorator.py

```diff
@@ -9,14 +10,26 @@
 from typing import Union
 
 import grpc
+from google.protobuf import any_pb2
 from google.protobuf.descriptor import FieldDescriptor
 from google.protobuf.message import Message
+from google.rpc import code_pb2
+from google.rpc import error_details_pb2
+from google.rpc import status_pb2
 from grpc_argument_validator import AbstractArgumentValidator
+from grpc_argument_validator import ArgumentValidatorConfig
 from grpc_argument_validator.argument_validators import NonDefaultValidator
 from grpc_argument_validator.argument_validators import NonEmptyValidator
 from grpc_argument_validator.argument_validators import UUIDBytesValidator
 from grpc_argument_validator.fields import validate_field_names
 from grpc_argument_validator.validation_context import ValidationContext
+from grpc_status import rpc_status
+
+
+@dataclass
+class _Error:
```

Comment:
Maybe we can make this more consistent with the protos and go for `_ErrorDetail` or `_FieldViolation`